### DISCUSSION TITLE
Deployment error on the basic Vercel blog

### DISCUSSION THREAD

**Discussion by AnesBkdm** [vercel/next.js#25512] [Help] [answered]
Hello, how are you?

Just finished playing with the base blog project, but I can't seem to deploy the project on Vercel. Everything works perfect locally, and when building for personal server. Tried looking out for dependencies etc. but nothing seems to move much. Anyone stumbled upon this problem? Thanks!

`

11:22:59 | Failed to compile.
-- | --
11:22:59 | ModuleNotFoundError: Module not found: Error: Can't resolve './styles/utils.module.css' in '/vercel/path0/pages'
11:22:59 | &gt; Build error occurred
11:22:59 | Error: &gt; Build failed because of webpack errors
11:22:59 | at /vercel/path0/node_modules/next/dist/build/index.js:17:924
11:22:59 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:6:584)

`

**Comment by leerob** [comment]
Hey! This appears to be a code-level error. Your build is failing because it can't find ` './styles/utils.module.css'`.

You should be able to verify by running `next build` locally.

**Comment by dankosykh** [comment, marked as answer]
Thanks for reaching out. Issue was due to the dependencies not being written to the package.json. It has been resolved. Also, thank you for your tutorial videos!

Thank you
Daniel Kosykh
